modules/rosa-sts-oidc-provider-command.adoc

```diff
@@ -32,11 +32,12 @@ When using `manual` mode, the `aws` command is printed to the terminal for your
 [source,terminal]
 ----
 aws iam create-open-id-connect-provider \
-	--url https://rh-oidc.s3.<aws_region>.amazonaws.com/<cluster_id> \
+	--url https://oidc.op1.openshiftapps.com/<oidc_config_id> \// <1>
 	--client-id-list openshift sts.<aws_region>.amazonaws.com \
-	--thumbprint-list <thumbprint> <1>
+	--thumbprint-list <thumbprint> <2>
 ----
-<1> The thumbprint is generated automatically when you run the `rosa create oidc-provider` command. For more information about using thumbprints with AWS Identity and Access Management (IAM) OpenID Connect (OIDC) identity providers, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc_verify-thumbprint.html[the AWS documentation].
+<1> The URL used to reach the OpenID Connect (OIDC) identity provider once the cluster is created.
```

Comment:
```suggestion
<1> The URL used to reach the OpenID Connect (OIDC) identity provider after the cluster is created.
```
Per the ISG, don't use "once" to mean "after": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o